.github/actions/replace_specs_snippets/action.yml

```diff
@@ -62,7 +62,10 @@ runs:
 
       - name: Generate Code Snippets
         run: |
-          node site/scripts/add-code-snippets-oas.js
+          cd site
+          npm install js-yaml
```

Comment:
Why are you installing one package here? Should it be included in a package.json file somewhere so that we can instead run `npm install` 

Maybe in the `scripts` directory we have a package.json and commands to run everything from within the scripts directory instead of having to cd into everything or run commands on nested directories